org.eclipse.xtext.xtext.ui.examples/projects/arithmetics/org.eclipse.xtext.example.arithmetics.ui/src-gen/org/eclipse/xtext/example/arithmetics/ui/internal/ArithmeticsActivator.java

```diff
@@ -34,7 +34,7 @@ public class ArithmeticsActivator extends AbstractUIPlugin {
 	
 	private static ArithmeticsActivator INSTANCE;
 	
-	private Map<String, Injector> injectors = Collections.synchronizedMap(Maps.<String, Injector> newHashMapWithExpectedSize(1));
+	private Map<String, Injector> injectors = Collections.synchronizedMap(new HashMap<>(2));
```

Comment:
i propose to revert this unrelated and regenerate ALL languages incl test languages in a 2nd step

Reply:
I agree: I'd only keep the changes to the UI modules that concern the fix of this PR.

Reply:
Generating all languages resulted in changes to over 150 files, which seems unrelated to this typo fix; many of them are like the one above. Please let me know if this should be the desired outcome.

Reply:
No I want you to revert all unrelated changes in your pr

Reply:
To make sure I understand what you are pointing to, shall I revert the unrelated changes only from this change or do I have to regenerate all the langues then revert the unrelated changes after?

Reply:
The changes to the 2 re-generated abstract ui modules must be kept, or the examples won't work.